BlazarService/src/main/java/com/hubspot/blazar/util/ModuleBuildLauncher.java

```diff
@@ -84,7 +87,7 @@ private BuildConfig fetchBuildpack(GitInfo gitInfo) throws IOException, NonRetry
   }
 
   private static BuildConfig mergeConfig(BuildConfig primary, BuildConfig secondary) {
```

Comment:
They're all collections so I think its a lot nicer to just use an empty collection rather than wrap in an optional